src/components/Buttons/LinkButton.tsx

```diff
@@ -10,10 +10,6 @@ const UnstyledLink = styled(Link)`
     display: contents;
 `;
 
-const UpperCaseButton = styled(Button)`
-    text-transform: uppercase;
```

Comment:
why is this deleted? I thought people wanted it uppercase (even if it doesn't _yet_ work on `prod` build!)

Reply:
Obviously correct me if I'm wrong on this, why couldn't we just type in uppercase if we want the button to be uppercase instead of having a separate component?